wirecard/oxid-ee
# WDPD-99 Show additional costs in PayPal basket

### This PR

* adds all additional costs or reductions (shipping, vouchers, wrapping, gift cards, payment costs) for an order to the PayPal basket overview

### How to Test

* log into the admin interface of the shop
* under "Shop Settings > Shipping Methods" make sure that the all shipping methods are assigned to all payment methods (under the "Payment" tab of each shipping method" there is the "Assign payment methods" button)
* under "Shop Settings > Payment Methods", assign a "Price Surcharge/Reduction" of more than 0 to the "Wirecard PayPal" payment method
* go to "Shop Settings > Coupon Series" and click on "Create new Coupon Serie"
* enter a name, set a currently valid date range, enter an amount (e.g. 10) for the discount and minimum order sum (e.g. 0) and click "Yes" on all the radio buttons, then click "Save"
* on the right side, create 10 new random coupons, click "Generate" and then "Export"
* now you are able to "Download" the coupons as a CSV file
* in the OXID shop frontend, create a new order with at least one item
* in the checkout process, redeem at least one of the generated coupon codes
* in the same step, add wrapping to at least one of the items
* in the same modal dialog, add a gift card to the order
* in the third step of the checkout process, select a shipping method that has costs greater than 0
* when you click on the total order sum on the PayPal checkout page, all items should be included translated to the current language of the shop user interface

### Jira Links

* https://jira.parkside.at/browse/WDPD-99
